charts/mirrorbits/values.yaml

```diff
@@ -85,6 +85,15 @@ tolerations: []
 
 affinity: {}
 
+geoipupdate:
+  image:
+    repository: maxmindinc/geoipupdate
+    tag: v4.3
+  #account_id:
```

Comment:
I think this needs to be set to empty for linting purposes.

Reply:
`1 chart(s) linted, 0 chart(s) failed`

the chart doesn't work with no values provided anyway...

needs azure keys and ingress to be configured iirc

Reply:
fair, I wasn't certain if you could b64enc nil. I know it complains when dicts are undefined